raiden/api/python.py

```diff
@@ -109,31 +137,32 @@ def flatten_transfer(transfer: LockedTransferState, role: str) -> typing.Dict[st
 
 
 def get_transfer_from_task(
-        secrethash: typing.SecretHash,
+        secrethash: SecretHash,
         transfer_task: TransferTask,
-) -> LockedTransferState:
-    transfer = None
+) -> Tuple[LockedTransferType, str]:
     role = views.role_from_transfer_task(transfer_task)
-
-    if role == 'initiator':
+    transfer: LockedTransferType
```

Comment:
I think this should be initialized `None`, because the lines 147-152 don't guarantee an assignment.

Reply:
Since transfer is returned, and is potentially returned as `None`, the caller(`transfer_tasks_view`) does check if for None value.
However, shouldn't the return type here be `Tuple[Optional[LockedTransferType], str]` in that case?